### ISSUE TITLE
Enable Microsoft Teams notifications

### ISSUE THREAD

**Issue by j771** [armory/spinnaker-operator#187]
Following this: https://spinnaker.io/setup/features/notifications/#microsoft-teams

I tried the following:

```
kind: SpinnakerService
metadata:
  name: spinnaker
spec:
  spinnakerConfig:
    profiles:
      echo:
        echo.yml: |
          microsoftteams:
            enabled: true
```

as well as these:

```
kind: SpinnakerService
metadata:
  name: spinnaker
spec:
  spinnakerConfig:
    profiles:
      echo:
        microsoftteams:
          enabled: true
```

```
kind: SpinnakerService
metadata:
  name: spinnaker
spec:
  spinnakerConfig:
    profiles:
      echo:
        echo-local.yml: |
          microsoftteams:
            enabled: true
```

None of them worked.

How do you enable Teams notifications in Spinnaker with the spinnaker operator?

**Comment by cristhian-castaneda** [comment]
Hey @j771 - thanks for open the issue.
Microsoft teams support will be released on **1.23** more info [here](https://github.com/spinnaker/spinnaker/issues/5826)

**Comment by cristhian-castaneda** [comment]
When the release happens, you can update spinnaker, and the settings to activate the Microsoft Teams feature will work without a problem.

**Comment by cristhian-castaneda** [comment]
@j771 here the release calendar https://spinnaker.io/community/releases/release-cadence/#upcoming-releases

**Comment by j771** [comment]
Awesome thanks!

So will this be the correct config?
```
kind: SpinnakerService
metadata:
  name: spinnaker
spec:
  spinnakerConfig:
    profiles:
      echo:
        echo.yml: |
          microsoftteams:
            enabled: true
```

**Comment by somyagarg94** [comment]
I am trying to setup my org MS Teams notifications via connector, I am seeing 404 error in the spin echo logs. 
I tested the webhook link locally, it works fine. 
`curl -H "Content-Type: application/json" -d "{\"text\": \"Hello World\"}" https://{orgname}.webhook.office.com/webhookb2/25eb8a6b-d551-4549-b006-597cb1711129@36da45f1-dd2c-4d1f-af13-5abe46b99921/IncomingWebhook/6d92176f12c84397bf97f7e8610e6c02/814d8929-b5b6-4483-xxxxx-xxxxxxxxxxx`

I noticed in the echo logs, it automatically converts the original org webhook URL into something like this: `https://outlook.office.com/webhook//webhookb2/25eb8a6b-d551-4549-b006-597cb1711129@36da45f1-dd2c-4d1f-af13-5abe46b99921/IncomingWebhook/8a066fa5e1ce4bb69d1c43daf9d81764/814d8929-b5b6-4483-xxxxx-xxxxxxxxxxx` and here I am seeing a 404 error. This seems to like a bug
